Sources/FigmaExport/Loaders/ColorsLoader.swift

```diff
@@ -37,29 +37,39 @@ final class ColorsLoader {
                                                                       lightHC: [Color]?,
                                                                       darkHC: [Color]?) {
         let colors = try loadColors(fileId: figmaParams.lightFileId, filter: filter)
-        
-        let darkSuffix = colorParams?.darkModeSuffix ?? "_dark"
+
+        let lightPrefix = colorParams?.lightModePrefix
+        let darkPrefix = colorParams?.darkModePrefix
+        let lightSuffix = colorParams?.lightModeSuffix
+        let darkSuffix = colorParams?.darkModeSuffix
```

Comment:
To maintain backward compatibility
```suggestion
        let darkSuffix = colorParams?.darkModeSuffix ?? "_dark"
```

Reply:
This makes no sense for me. Someone can use suffix, someone can use prefix, but not both. So if I only have suffixes in my Figma, how would I configure my figma-export with such default values?